repository-data/application/src/main/resources/hcm-config/namespaces/externalstorage.cnd

```diff
@@ -0,0 +1,9 @@
+<jcr='http://www.jcp.org/jcr/1.0'>
```

Comment:
Why using a different namespace? I think is possible to use the common: then it's easier to inject this field in specific document types

Reply:
this will be exported and package as a generic plugin, hence new namespace.

in doc type instead of:

```yaml
  ...
  hipposysedit:type: hippo:resource
  ...
```

you will have 

```yaml
  ...
  hipposysedit:type: externalstorage:resource
  ...
```